agent/__init__.py

```diff
@@ -524,6 +524,7 @@ def clear_credentials(path):
 
         if not os.path.exists(CREDENTIALS_PATH):
             os.mkdir(CREDENTIALS_PATH, 0o711)
+        os.chmod(CREDENTIALS_PATH, 0o711)
```

Comment:
Shouldn't this be an 'else' here? kinda redundant otherwise.

Reply:
it would have any sense only once.

and this issue happens only cause upgrade

Reply:
fixed